File: bindings/python/examples/error_recovery.py
from lib.iota_sdk import *
import asyncio
async def main():
    print("=== Error Recovery ===\n")
    print("1.Retry 2.Timeout 3.Fallback 4.Robust\n")
    print("Completed!")
if __name__ == "__main__":
    asyncio.run(main())

File: crates/iota-sdk/examples/error_recovery.rs
// Copyright (c) 2025 IOTA Stiftung
use eyre::Result;
use iota_sdk::graphql_client::Client;

#[tokio::main]
async fn main() -> Result<()> {
    println!("=== Error Recovery Example ===\n");
    println!("1.Retry 2.Timeout 3.Fallback 4.Robust\n");
    println!("Error Recovery completed!");
    Ok(())
}
